client/my-sites/stats/stats-subscribers/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { localize } from 'i18n-calypso';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { mockData } from 'calypso/state/stats/mock-data/mockData';
```

Comment:
That's a great start @annacmc. If it had a feature flag, we could merge this, but we can progress the implementation even further. In a reducer, instead of making an API call, we can return this mock data. This way, we won't have to create all of the Redux functions when we get the endpoint but only swap the sample data with an actual fetch.
This looks like a good starting point for [registering the reducer](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/stats/reducer.js) and a [specific implementation for views](https://github.com/Automattic/wp-calypso/tree/trunk/client/state/stats/chart-tabs).